pset-iterators.js

```diff
@@ -183,6 +215,20 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
+const currentYear = 2020
+const shortTermPresidents = presidents.filter((president) => {
+  let enterYear = parseInt(president.took_office.split('-')[0])
+  let exitYear
+  if (president.left_office === null){
+    exitYear = currentYear
```

Comment:
Way to consider edge cases here! For good practice, be careful naming a variable outside of the function scope as in line 218 (unless you intended it to be a global variable).